runtimes/common/src/constants.rs

```diff
@@ -297,6 +297,8 @@ pub mod governance {
 		pub const TechnicalMotionDuration: BlockNumber = TECHNICAL_MOTION_DURATION;
 		pub const TechnicalMaxProposals: u32 = 100;
 		pub const TechnicalMaxMembers: u32 = 100;
+		// Tipper Group
+		pub const TipperMaxMembers: u32 = 21;
```

Comment:
Didn't we say this would be 7? Has anything changed since then?

Reply:
The variable `TipperMaxMembers` is just defining the max amount of tippers. The council can still just select 7 tippers. I changed it to 21, since in the near or far future, we want to select more tippers, and then we do not have to change the source code again.